cli/src/cmd/forge/init.rs

```diff
@@ -70,18 +70,33 @@ impl Cmd for InitArgs {
         }
         let root = dunce::canonicalize(root)?;
 
-        // if a template is provided, then this command is just an alias to `git clone <url>
-        // <path>`
+        // if a template is provided, then this command clones the template repo, removes the .git
+        // folder, and initializes a new git repo—-this ensures there is no history from the
+        // template and the template is not set as a remote.
         if let Some(template) = template {
             let template = if template.starts_with("https://") {
                 template
             } else {
                 "https://github.com/".to_string() + &template
             };
             p_println!(!quiet => "Initializing {} from {}...", root.display(), template);
+
             Command::new("git")
                 .args(["clone", "--recursive", &template, &root.display().to_string()])
                 .exec()?;
+
```

Comment:
don't we also have to "cd" into it (std::env::set_current_dir)?

Reply:
ah yes it seems `root` can be set and is not always the current dir, will fix

https://github.com/foundry-rs/foundry/blob/7518d37b586023be3c5f7f5eeb7a53584da7ae70/cli/src/cmd/forge/init.rs#L66

Reply:
should be fixed in 75f918a1

Reply:
I think that .current_dir on the clone command was fine, i meant that when we're deleting the .git folder and initializing, we have to move into in the newly cloned repo directory